### ISSUE TITLE
Missing type "updateUnreadMessageCount"

### ISSUE THREAD

**Issue by JackD111** [x2bool/tdsharp#4]
Hi, 

the type class for "updateUnreadMessageCount" was missing, I had to add it manually to prevent converter crashes. On a quick glance, it seems the types are incomplete and your missing around 300+ types :>

```
using Newtonsoft.Json;

namespace TD {

    public partial class Update : Structure {

        public class UpdateUnreadMessageCount : Update {

            [JsonProperty( "@type" )]
            public override string DataType { get; set; } = "updateUnreadMessageCount";

            [JsonProperty( "@extra" )]
            public override string Extra { get; set; }

            [JsonConverter( typeof( Converter ) )]
            [JsonProperty( "unread_count" )]
            public int UnreadCount { get; set; }

            [JsonConverter( typeof( Converter ) )]
            [JsonProperty( "unread_unmuted_count" )]
            public int UnreadUnmutedCount { get; set; }
        }

    }

}
```

**Comment by x2bool** [comment]
Hi! It seems this type was added recently: https://github.com/tdlib/td/commit/0c9249f7f63b57c599bc98b3528a351547fb6f57 along with many others. I will regenerate the types. Thanks for pointing out this issue.

**Comment by x2bool** [comment]
Fixed with fe8b02e93c7482a69508d454fd5e41bad8262e66

 https://www.nuget.org/packages/TDLib/1.2.0
